angular-client/src/pages/landing-page/components/landing-page-latency-display/landing-page-latency-display.component.html

```diff
@@ -0,0 +1,32 @@
+<info-background>
+  <div class="latency-display">
+    <div class="latency-display-header">
+      <typography variant="header" content="Latency" />
+    </div>
+
+    <div class="latency-display-body">
+      <typography
+        variant="subheader"
+        [content]="latency.toString()"
+        additionalStyles="padding-right: 2px; margin-top: 0px; margin-bottom: 0px;"
+      />
+      <typography variant="info-subtitle" content="ms" additionalStyles="font-size: 15px;" />
+    </div>
+  </div>
+</info-background>
+
+<!-- <vstack>
```

Comment:
unused

Reply:
honestly delete these latency display components if we're not using them anymore